src/NSwag.CodeGeneration.CSharp/SwaggerToCSharpControllerGeneratorSettings.cs

```diff
@@ -53,5 +55,11 @@ public string GetRouteName(SwaggerOperation operation)
 
         /// <summary>Gets or sets the strategy for naming routes (default: CSharpRouteNamingStrategy.None).</summary>
         public CSharpControllerRouteNamingStrategy RouteNamingStrategy { get; set; }
+
+        /// <summary>Gets or sets a value indicating whether to allow adding model validation attributes</summary>
+        public bool UseModelValidationAttributes { get; set; }
```

Comment:
Maybe rename to GenerateModelValidationAttributes or GenerateValidationAttributes ?

Reply:
I ve renamed to GenerateModelValidationAttributes